app/views/search/_placeholders.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="col-sm-<%= box_width %>">
+  <h2><%= heading %></h2>
+  <div id="<%= id %>">
```

Comment:
Ah, got it. I noticed the one you added for worldcat didn't have those so I removed it from all of them not knowing what they were for in the first place :)

Thanks for the explanation and I'll add those back in.